gapic-generator/templates/default/lib/_service.erb

```diff
@@ -1,4 +1,21 @@
 <%- assert_locals service -%>
+
+<%- if service.generate_rest_clients? -%>
+<% @requires = capture do %>
+require "gapic/config"
+require "gapic/config/method"
```

Comment:
Do we not need gapic/common here?

This section also seems mostly the same as the non-rest case below. Just the end (lro and client_require vs service_rest_require) are different.

Reply:
gapic/common brings grpc in immediately (